apps/framework-cli/src/framework/core/infrastructure_map.rs

```diff
@@ -140,11 +137,53 @@ impl TopicToTableSyncProcess {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
+pub enum TopicChange {
+    Added(Topic),
+    Removed(Topic),
+    Updated { before: Topic, after: Topic },
+}
```

Comment:
Looks like these enum can be generic.

Reply:
Yes they can - thanks for the call out that was an easy change